sdcm/tester.py

```diff
@@ -678,6 +680,7 @@ def configure_ldap(self, node, use_ssl=False):
                                           user=ldap_username, password=LDAP_PASSWORD, ldap_entry=user_entry)
 
     def _init_test_timeout_thread(self) -> threading.Timer:
+        self.start_time = time.time()
```

Comment:
why moving this one ? it's used in more places then just this function....

Reply:
yes, but _init_tets_timeout_thread is used during test initialization, so it's updated anyway.
Resetting start_time is important as test is restarted, otherwise timeout would occur.
From the other hand, anyway need to adjust test_duration in config file as jenkins would kill test anyway.

So I'd need to fix also 'get_test_duration' to make it fully working.

Reply:
I would skip that change, and change the test_duration for 12h as a starter.

and if we really would need such a case that can constantly run and extend it's timeout, we'll need to design it and consider how to implement, now we don't need such a thing.